### PR TITLE
Feature/DTGB-288: Added CTA block styling + refactor of boxes code.

### PR DESCRIPTION
**Pull request by meirege**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.

### CODE REVIEW

File: styleguide/components/21-atoms/link/_link.scss [hunk 1]
```diff
@@ -36,6 +36,22 @@ a {
       }
     }
 
+    &[href*="mailto:"] {
```
Comment (Jeroen005) [inline styleguide/components/21-atoms/link/_link.scss]: Hier mag een ^= staan ipv *=

File: styleguide/components/21-atoms/link/_link.scss [hunk 2]
```diff
@@ -130,6 +146,10 @@ a {
     @include icon(download, after);
   }
 
+  &[href*="mailto:"] {
```
Comment (Jeroen005) [inline styleguide/components/21-atoms/link/_link.scss]: same here

File: styleguide/components/41-organisms/search-block/_search-block.scss [hunk 3]
```diff
@@ -1,32 +1,5 @@
...
-
-    @include tablet {
-      align-self: center;
-      width: 8rem;
-      height: 8rem;
-      margin: 0 2.5rem 0 0;
-      justify-self: flex-start;
-    }
-  }
+@include box-left;
```
Comment (Jeroen005) [inline styleguide/components/41-organisms/search-block/_search-block.scss]: indentation is niet correct denk ik

File: styleguide/components/_preview.twig [hunk 4]
```diff
@@ -6,6 +6,7 @@
       <title>{% block title %}Preview Layout{% endblock %}</title>
 
       <link media="all" rel="stylesheet" href="{{ '/styleguide/vendor/baguettebox/dist/baguetteBox.min.css'| path }}">
+          <link rel="stylesheet" href="https://openlayers.org/en/v5.0.0/css/ol.css" type="text/css">
```
Comment (Jeroen005) [inline styleguide/components/_preview.twig]: indent

File: templates/contrib/paragraphs/cta/paragraph--call-to-action.html.twig [hunk 5]
```diff
@@ -0,0 +1,51 @@
+{% extends "paragraph.html.twig" %}
+
+ {% block paragraph %}
```
Comment (Jeroen005) [inline templates/contrib/paragraphs/cta/paragraph--call-to-action.html.twig]: indent

**Review by Jeroen005 — CHANGES_REQUESTED** [review]

File: styleguide/components/21-atoms/link/_link.scss (hunk 1 above)
Comment (meirege) [reply styleguide/components/21-atoms/link/_link.scss]: Fixed.

File: styleguide/components/21-atoms/link/_link.scss (hunk 2 above)
Comment (meirege) [reply styleguide/components/21-atoms/link/_link.scss]: Fixed.

File: styleguide/components/41-organisms/search-block/_search-block.scss (hunk 3 above)
Comment (meirege) [reply styleguide/components/41-organisms/search-block/_search-block.scss]: Fixed.

File: styleguide/components/_preview.twig (hunk 4 above)
Comment (meirege) [reply styleguide/components/_preview.twig]: Fixed.

File: templates/contrib/paragraphs/cta/paragraph--call-to-action.html.twig (hunk 5 above)
Comment (meirege) [reply templates/contrib/paragraphs/cta/paragraph--call-to-action.html.twig]: Fixed.

**Review by Jeroen005 — APPROVED** [review]

**Pull request merged**